standard/classes.md

```diff
@@ -4413,7 +4418,12 @@ Each of the types referenced in the *formal_parameter_list* of an instance const
 
 The optional *constructor_initializer* specifies another instance constructor to invoke before executing the statements given in the *constructor_body* of this instance constructor. This is described further in [§14.11.2](classes.md#14112-constructor-initializers).
 
-When a constructor declaration includes an `extern` modifier, the constructor is said to be an ***external constructor***. Because an external constructor declaration provides no actual implementation, its *constructor_body* consists of a semicolon. For all other constructors, the *constructor_body* consists of a *block*, which specifies the statements to initialize a new instance of the class. This corresponds exactly to the *block* of an instance method with a `void` return type ([§14.6.11](classes.md#14611-method-body)).
+When a constructor declaration includes an `extern` modifier, the constructor is said to be an ***external constructor***. Because an external constructor declaration provides no actual implementation, its *constructor_body* consists of a semicolon. For all other constructors, the *constructor_body* consists of either
+
+- a *block*, which specifies the statements to initialize a new instance of the class.
+- or an expression body, which consists of `=>` followed by an *expression* and a semicolon, and denotes a single expression to initialize a new instance of the class.
+
+A *constructor_body* that is a *block* or expression body corresponds exactly to the *block* of an instance method with a `void` return type ([§14.6.11](classes.md#14611-method-body)).
```

Comment:
This was effectively present before, but I'm not entirely sure what means by "corresponds exactly to" here... should we clarify it, while we're reworking this bit? (I suspect it's in terms of what statements or expressions are valid...)

Reply:
Given that this isn't specific to expression-bodied members, I'll raise a separate issue about that.

Reply:
(Also, Eric noted that this isn't the same meaning of "corresponds" as "A get accessor corresponds to a parameterless method...")

Reply:
Raised #577/